CONTRIBUTING.md

```diff
@@ -1,227 +1,150 @@
 ```
 
-> ⚠️  If the installation failed, please remove the global packages related to Strapi. The command `npm ls strapi` will help you to find where your packages are installed globally.
+#### 3. ⏳ Install the dependencies
 
-#### 4. 🏗 Create a new project
+Go to the root of the repository.
 
-You can open a new terminal window and go into any folder you want for the next steps.
 ```bash
-cd ../workspace/
+cd strapi && yarn setup
 ```
 
-The command to generate a project is the same, except you have to add the `--dev` argument at the end of line.
-```bash
-strapi new my-project --dev
-```
+#### 4. 🚀 Start the example application
 
-#### 5. 🚀 Start the project
+**Go to the getstarted application**
 
-First, you have to start the server.
 ```bash
-cd ./my-project
-strapi start
+cd strapi/examples/getstarted
+yarn develop
 ```
 
 The server (API) is available at http://localhost:1337
 
 > ⚠️  If you've followed the recommended setup, you should not be able to reach the administration panel at http://localhost:1337/admin.
 
-Then, you have to start the Webpack server to build and run the administration.
+**Start the administration panel server**
+
 ```bash
-cd ./my-project/admin
-npm run start
+cd strapi/packages/strapi-admin
+yarn develop
 ```
 
 The administration panel is available at http://localhost:4000/admin
 
 **Awesome! You are now able to contribute to Strapi.**
 
----
-
-## Plugin Development Setup
-
-To create a new plugin, you'll have to run the following commands:
-
-#### 1. 🏗 Generate a new plugin
-
-```bash
-cd ./my-project
-strapi generate:plugin my-plugin
-```
-
-#### 2. ✅ Verify the symlink
-
-Make sure that the `strapi-helper-plugin` is linked to your project.
-
-Please run this command in the repository folder where Strapi is cloned:
-```bash
-cd /repository/strapi/packages/strapi-helper-plugin
-npm link
-```
-
-Link the `strapi-helper-plugin` node_modules in the plugin folder:
-```bash
-cd ./my-project/plugins/my-plugin
-npm link strapi-helper-plugin
-```
-
-#### 3. 🚀 Start the project
-
-```bash
-cd ./my-project/admin
-npm run start
-```
-
-The administration panel is available at http://localhost:4000/admin
+#### 5. Available commands
+
+- `yarn watch` starts yarn watch in all packages.
+- `yarn build` builds the `strapi-helper-plugin` (use this command when you develop in the administration panel).
+- `yarn setup` installs the dependencies.
+- `yarn lint` lints the codebase.
+- `yarn test:clean` removes the coverage.
+- `yarn test:front` runs the front-end related tests.
+- `yarn test:front:watch` runs an interactive test watcher for the front-end.
+- `yarn test:snyk` checks the dependencies vulnerabilities.
+- `yarn test:unit` runs the back-end unit tests.
+- `yarn test:e2e` runs an end-to-end test suite.
+- `yarn test:generate-app` generates a test application.
+- `yarn test:start-app` starts the test application.
 
 ---
 
 ## Running the tests
 
 You can run three different kind of tests:
```

Comment:
This entire sentence doesn't make sense anymore.  There is only a section about databases.  Perhaps the "Running the tests" is also incorrectly labelled?